core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v1/RecipeEndpoint.java

```diff
@@ -83,20 +53,6 @@
             nickname = "deleteRecipe")
     void delete(@PathParam("id") Long id);
 
-    @DELETE
-    @Path("account/{name}")
-    @Produces(MediaType.APPLICATION_JSON)
-    @ApiOperation(value = RecipeOpDescription.DELETE_PUBLIC_BY_NAME, produces = ContentType.JSON, notes = Notes.RECIPE_NOTES,
-            nickname = "deletePublicRecipe")
```

Comment:
like above can we keep the same API just change the service layer? E.g. deletePublicRecipe and deletePrivateRecipe should call the same service and the implemhtation of this two shall be identical.

This comment is valid for every method in this interface

Reply:
yepp.